packages/babel-core/src/config/files/configuration.js

```diff
@@ -24,6 +24,30 @@ const BABELRC_FILENAME = ".babelrc";
 const BABELRC_JS_FILENAME = ".babelrc.js";
 const BABELIGNORE_FILENAME = ".babelignore";
 
+export function findClosestConfig(rootDir: string): string {
```

Comment:
Instead of `Closes`, which is not very accurate, could we name this: `findConfigUpwards`, `findParentConfig`, etc.

Reply:
Those are good suggestions, thanks. I'll go with `findConfigUpwards`.